src/layer/vector/Polygon.js

```diff
@@ -84,6 +84,34 @@ L.Polygon = L.Polyline.extend({
 		return this._map.layerPointToLatLng(center);
 	},
 
+	// @method contains (latlng: LatLng): Boolean
+	// Returns `true` if `latlng` is inside the (multi)polygon.
+	contains: function (latlng) {
```

Comment:
Leaflet API usually also allows a simple array as `latlng` argument, shouldn't this function also allow that by adding a `latlng = L.latlng(latlng);` line?